manual/src/cmds/native-debugger.etex

```diff
@@ -0,0 +1,289 @@
+\chapter{Native debugger} \label{c:native-debugger}
+%HEVEA\cutname{native-debugger.html}
+
+\section{s:native-debugger-preliminaries}{Preliminaries}
+
+This chapter describes the support for debugging executables built with \texttt{ocamlopt}, the native-code compiler, using GDB or LLDB on Linux, macOS, and FreeBSD. We will call this \texttt{native debugging}, in contrastto bytecode debugging supported via \texttt{ocamldebug} (see chapter~\ref{c:debugger}).
```

Comment:
```suggestion
This chapter describes the support for debugging executables built with \texttt{ocamlopt}, OCaml's native compiler, using GDB or LLDB on Linux, macOS, or FreeBSD. We will call this \texttt{native debugging}, in contrastto bytecode debugging supported via \texttt{ocamldebug} (see chapter~\ref{c:debugger}).
```
Let me know if these suggestions change the meaning.

Reply:
Drive by comment: `contrastto` -> `contrast to`